assets/stylesheets/style.scss

```diff
@@ -1,85 +1,150 @@
 $green: #1FB682;
 
+//GENERAL
+
+html {
+  overflow: auto
+}
+
+body {
+  height: 100%;
+  overflow: hidden;
+  width: 100%;
+}
+
 * {
+	-moz-osx-font-smoothing: grayscale;
```

Comment:
I think some of these should probably be set on the `body` rather than `*`. Any reason you chose to do it on `*`? (Sorry if this was here before and I just didn't catch it last time around)

Also, the indenting here is off.